<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mask_manager</class>
 <widget class="QDialog" name="mask_manager">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mask Management</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1" colspan="2">
    <widget class="QPushButton" name="view_masks">
     <property name="text">
      <string>View Masks</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QPushButton" name="show_all_masks">
     <property name="text">
      <string>Show All Masks</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" rowspan="2" colspan="2">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QPushButton" name="hide_all_boundaries">
     <property name="text">
      <string>Hide All Boundaries</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QPushButton" name="export_masks">
     <property name="text">
      <string>Export Masks</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QPushButton" name="border_color">
     <property name="text">
      <string>Border Color</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QPushButton" name="panel_buffer">
     <property name="text">
      <string>Masks to Panel Buffer</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="12">
    <widget class="QTreeWidget" name="masks_tree">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Presentation</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Remove</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QPushButton" name="show_all_boundaries">
     <property name="text">
      <string>Show All Boundaries</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QPushButton" name="import_masks">
     <property name="text">
      <string>Import Masks</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QPushButton" name="hide_all_masks">
     <property name="text">
      <string>Hide All Masks</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QPushButton" name="apply_changes">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Apply Changes</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>masks_tree</tabstop>
  <tabstop>panel_buffer</tabstop>
  <tabstop>import_masks</tabstop>
  <tabstop>export_masks</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
